e2e/Modals.test.js

```diff
@@ -21,7 +21,17 @@ describe('modal', () => {
     await expect(elementById(testIDs.WELCOME_SCREEN_HEADER)).toBeVisible();
   });
 
-  it('show multiple modals', async () => {
+  it('unmount modal when dismissed', async () => {
```

Comment:
Please limit this test to Android is it fails on iOS. We'll enable it for both platforms once it's fixed on iOS.
To do so, change ` it('unmount modal when dismissed', async () => {` to `it(':android: unmount modal when dismissed', async () => {`